tutorials/modules.html

```diff
@@ -0,0 +1,646 @@
+<div class="static-content">
+  <div class="index-list">
+    <p>This page presents a guide on using modules.</p>
+    <ul>
+      <li><a href="#introduction">Usefulness of modules</a></li>
+      <li><a href="#how-to-use">How to use modules</a></li>
+      <li><a href="#importing-plugins">Importing plugins</a></li>
+      <li><a href="#importing-editors">Importing editors</a></li>
+      <li><a href="#importing-renderers">Importing renderers</a></li>
+      <li><a href="#importing-validators">Importing validators</a></li>
+      <li><a href="#importing-cell-types">Importing cell types</a></li>
+      <li><a href="#importing-locales">Importing locales</a></li>
+      <li><a href="#moment-js-optimize">Optimizing moment.js locales</a></li>
+      <li><a href="#using-with-wrappers">Using with wrappers</a></li>
+      <li><a href="#tree-shaking">Tree shaking</a></li>
+      <li><a href="#modules-cheatsheet">Modules cheatsheet</a></li>
+
+    </ul>
+  </div>
+
+  <div class="example-container clearfix">
+    <h3 id="introduction">Usefulness of modules</h3>
+
+    <p>
+      The full bundle of Handsontable consists of multiple built-in modules such as plugins, renderers, editors, validators, and cell-types.
+      It also includes the whole rendering functionality and advanced data management. This is a compact tool with plenty of options available.
+    </p>
+    <p>
+      When you become familiar with Handsontable functionalities you may find out that some parts of it are essential in your application and some
+      of them stay idle most of the time.
+    </p>
+    <p>
+      Thanks to <strong><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules">modules</a></strong> the package could be split into
+      smaller pieces and you can <strong>import</strong> them as you need. Essentialy it was divided into two: the base and the optional part.
+    </p>
+    <p>
+      The base of Handsontable holds mandatory parts packed inside <code>handsontable/base</code>, and it includes vital parts for the component
+      to run. The rest is customizable based on what you want to import.
+    </p>
+    <p>
+     A mindful use of the modules brings a lot of optimization to the application, yet, it only needs several lines of code.
+    </p>
+    <p>
+      The graph presents a comparison of size in KB for a full bundle, basic optimization and with optimized locales. The sample code
+      is avaiable just below - it shows sample countries and cities and although it looks small it will generate over 345 KB (Gzipped):
+    </p>
+      <img src="images/modularization_comparison.png" alt="modularization_comparison_graph" />
+    <p>
+     You can compare the following examples to see the difference in KB of the final build.
+     Note: this is an example in a nutshell, the next section shows how to do it step by step.
+    </p>
+     <pre><code class="lang-js">
```

Comment:
And add the settings somewhere here:

```suggestion
    </p>
    
<pre><code class="lang-js">
  const settings =  {
  colHeaders: true,
  filters: true,
  data: [
    {
      city: "Fontainebleau",
      country: "France",
    },
    {
      city: "Milton",
      country: "United Kingdom",
    },
    {
      city: "Giedlarowa",
      country: "Poland",
    },
    {
      city: "Forssa",
      country: "Finland",
    },
    {
      city: "Halle",
      country: "Germany",
    }
  ],
  dropdownMenu: true,
  columns: [
    { data: 'city' },
    {
      data: 'country',
      type: 'dropdown',
      source: ['Finland', 'France', 'Germany', 'Poland', 'United Kingdom'],
    },
  ]
}
    </code></pre>
    
    <p>As you can see, few plugins are used here: filter, dropdownMenu. We can import default Handsontable class which will include full bundle and they will be included:</p>
    
     <pre><code class="lang-js"> ... </code></pre>
     
     <p>But this will load more plugins that we don;t need. Or we can use `handsontable/base` and load only the plugins that ... 
```

WDYT?

Reply:
Good idea, less noise